numerics/elliptic_integrals.cpp

```diff
@@ -45,39 +45,61 @@ double EllipticNomeQ(double mc);
 // where m = 1 - mc.  The method is similar to that described in [Fuk11a].
 void FukushimaEllipticBD(double mc, Angle& B_m, Angle& D_m);
 
+// The functions that compute elliptic integrals of multiple kinds may be called
+// without computing the integral of the third kind.  Passing |unused| bypasses
+// the corresponding code path at compile time, passing a reference to an
+// |Angle| performs the computation.
+
+struct UnusedResult {
+  constexpr UnusedResult(base::uninitialized_t) {}
+};
+
+template<typename T, typename = EnableIfAngleResult<T>>
+inline constexpr bool should_compute = !std::is_same_v<T, UnusedResult const>;
```

Comment:
Do we believe that template variables work on Mac?

Reply:
Seems to work.